planning_domains_api.py

```diff
@@ -128,27 +155,27 @@ def get_collections(ipc = None):
 
 def get_collection(cid):
     """Return the collection of a given id"""
-    return simple_query("/classical/collection/%d" % cid)
+    return simple_query("/collection/%d" % cid)
 
 def find_collections(name):
     """Find the collections matching the string name"""
-    return simple_query("/classical/collections/search?collection_name=%s" % name)
+    return simple_query("/collections/search?collection_name=%s" % name)
 
 def update_collection_stat(cid, attribute, value, description):
     """Update the attribute stat with a given value and description"""
     update_stat('collection', cid, attribute, value, description)
 
 def tag_collection(cid, tagname):
```

Comment:
Even methods like this should take the formalism as a parameter to pass on to `simple_query`

Reply:
Should I default simple_query's formalism to classical as well and then remove the /classical from the lines that call simple_query?

Reply:
Maybe leave simple query as requiring the parameter.

Reply:
![image](https://user-images.githubusercontent.com/71029388/231926222-fa9df150-1b50-4d8b-881e-59a5d517d0c2.png)
Should I add a formalism parameter to this get_collections function?

Another question: If I make simple_query require a formalism, for functions like this:
![image](https://user-images.githubusercontent.com/71029388/231926706-caaed4e8-e250-4c8f-b2cf-cedba628aa89.png)
would there still be a "/formalism" in front of the "/tags"? If not, then I'll change these functions to have formalism = "" (when calling simple_query)

Reply:
So yes to the get_collections question.

As for the tagging, it's not a clear-cut case. Recall this is what our db map looks like:

![image](https://user-images.githubusercontent.com/225019/231928899-dcecb95a-2719-4533-b088-dc0ff879475a.png)

An argument can be made either way, but let's stick with being consistent and require the formalism. It may become useful for authentication (some may be allowed to tag domains only form a formalism they are an expert/maintainer of).